aws/runExpoAppPublish.sh

```diff
@@ -289,9 +293,13 @@ function main() {
 
   # Create master export 
   info "Creating master OTA artefact with Extra Dirs: ${EXPO_EXPORT_MERGE_ARGUMENTS}"
-  expo export --public-url "${PUBLIC_URL}" --output-dir "${SRC_PATH}/app/dist/master/" ${EXPO_EXPORT_MERGE_ARGUMENTS}  || return $?
+  expo export --dump-sourcemap --public-url "${PUBLIC_URL}" --output-dir "${SRC_PATH}/app/dist/master/" ${EXPO_EXPORT_MERGE_ARGUMENTS}  || return $?
```

Comment:
Probably don't need the master export for the source maps. The master export is only run to generate an expo client manifest with multiple versions of expo installed. Its not used for a real mobile app